lib/src/model.dart

```diff
@@ -27,26 +27,34 @@ import 'model_utils.dart';
 import 'package_meta.dart' show PackageMeta, FileContents;
 import 'utils.dart';
 
-Map<String, Map<String, List<Map<String, dynamic>>>> __crossdartJson;
+Map<String, Map<String, Map<String, List<Map<String, dynamic>>>>> __crossdartJson = {};
```

Comment:
These generics are pretty deeply nested - it's difficult to know what the data structure is describing. Can you convert this to custom class(es) that wrap the nested types?